src/core/src/sys/cells.rs

```diff
@@ -174,6 +174,23 @@ impl UnitCell {
         return (x, y, z);
     }
 
+    /// Get a vector of the distances between faces of the unit cell
+    pub fn lengths_as_vec(&self) -> Vec<f64> {
```

Comment:
Good catch! I will change that. Actually, I tried to get the maximal value out of the tuple but it was kind of messy. I am not used to the fact that there is no `max` function for arrays and tuples.